File: kattis/bog/chapter1/easy/drinking_song/main.cpp
#include <bits/stdc++.h>
using namespace std;

int n;
string s;

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);
	cin >> n >> s;
	n++;
	while (--n) {
		cout << n << "bottles of milk on the wall, " << n 
		<< "bottles of milk.\n";
		if (n > 1)
			cout << "Take one down, pass it around, " << n-1
			<< (n-1 == 1 ? "bottle" else "bottles")
	}
}

File: kattis/bog/chapter1/easy/drinking_song/main.py
n = int(input())
s = input()
for i in range(n, 0, -1):
	if i > 2:
		print(f"{i} bottles of {s} on the wall, {i} bottles of {s}.\nTake one down, pass it around, {i-1} bottles of {s} on the wall.\n")
	elif i == 2:
		print(f"2 bottles of {s} on the wall, 2 bottles of {s}.\nTake one down, pass it around, 1 bottle of {s} on the wall.\n")
	else:
		print(f"1 bottle of {s} on the wall, 1 bottle of {s}.\nTake it down, pass it around, no more bottles of {s}.")
	
